UI/forms/OBSBasic.ui

```diff
@@ -215,7 +215,7 @@
       <property name="minimumSize">
```

Comment:
Could this property get removed altogether?

Reply:
Per off-thread discussion, this can be addressed at a later time.